doc/user/bgp.rst

```diff
@@ -2573,6 +2573,20 @@ are reached using *core* MPLS labels which are distributed using LDP or BGP
 labeled unicast.  *bgpd* also supports inter-VRF route leaking.
 
 
+L3VPN over GRE interfraces
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to configure and assign an incoming route-map to L3VPN peer,
+that permits BGP updates to be installed over a GRE interface instead of an
+MPLS based backbone. Command under route-map is the following one:
+
+.. clicmd:: set l3vpn next-hop encapsulation gre
+
+Acceptance of the route happens at BGP nexthop tracking resolution. For all
+incoming BGP updates, the resulting outgoing interface will be checked against
+interface type. Detection of a GRE interface will authorise to install this
+route, even if no MPLS backbone is available.
```

Comment:
I think rephrasing this from the user perspective would be helpful.  Again, unclear to me why MPLS is being mentioned.